packages/contractkit/src/utils/phone-number-lookup/bls-blinding-client.ts

```diff
@@ -0,0 +1,48 @@
+import { randomBytes } from 'crypto'
+
+export interface BlsBlindingClient {
+  blindMessage: (base64PhoneNumber: string) => Promise<string>
+  unblindAndVerifyMessage: (blindedMessage: string) => Promise<string>
+}
+
+export class WasmBlsBlindingClient implements BlsBlindingClient {
+  private thresholdBls: any
```

Comment:
We could make an interface for this, something that has a `blind` and `unblind` method.

Reply:
That's what `BlsBlindingClient` is. Did you have something different in mind?

Reply:
Fair enough, resolving.